d3-box/index.d.ts

```diff
@@ -3,9 +3,9 @@
 // Definitions by: Linkun Chen <https://github.com/lk-chen>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference path="../d3/d3.d.ts"/>
```

Comment:
The code compiles without an explicit reference -- that's pulled in by the import.